v63004/inref/gtm8874.m

```diff
@@ -0,0 +1,209 @@
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	WRITE "VIEW ""STATSHARE"":""DEFAULT""",!
+	VIEW "STATSHARE":"DEFAULT"
+	WRITE !
+	do checkRegion()
+	WRITE !
+	WRITE "VIEW ""NOSTATSHARE"":""AREG""",!
+	VIEW "NOSTATSHARE":"AREG"
+	WRITE !
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	WRITE "VIEW ""STATSHARE"""
+	VIEW "STATSHARE"
+	WRITE !
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	WRITE "VIEW ""NOSTATSHARE"""
+	VIEW "NOSTATSHARE"
+	WRITE !
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	quit
+
+
+
+testB1
+	WRITE "----------",!
+	WRITE "testB1",!
+	WRITE "----------",!!
+
+	WRITE "VIEW ""STATSHARE"""
+	VIEW "STATSHARE"
+	WRITE !
+	do checkRegion()
+	WRITE !
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	quit
+
+testB2
+	WRITE "----------",!
+	WRITE "testB2",!
+	WRITE "----------",!!
+
+	WRITE "STATSHARE DEFAULT: "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+	WRITE "STATSHARE AREG:    "
+	WRITE $VIEW("STATSHARE","AREG"),!
+	WRITE "STATSHARE BREG:    "
+	WRITE $VIEW("STATSHARE","BREG"),!
+	WRITE !
+
+	quit
+
+
+testC
+	WRITE "----------",!
+	WRITE "testC",!
+	WRITE "----------",!!
+
+	WRITE "SET $ZGBLDIR=""otherA.gld"" ",!
+	SET $ZGBLDIR="otherA.gld"
+
+	WRITE "VIEW ""STATSHARE"":""DEFAULT"" ",!
+	VIEW "STATSHARE":"DEFAULT"
+	WRITE !
+	do checkRegion()
+	WRITE !
+	WRITE "$VIEW(""STATSHARE"",""DEFAULT""): "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!!
+
+	WRITE "VIEW ""NOSTATSHARE"":""DEFAULT"" ",!
+	VIEW "NOSTATSHARE":"DEFAULT"
+	WRITE !
+	do checkRegion()
+	WRITE !
+	WRITE "$VIEW(""STATSHARE"",""DEFAULT""): "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!!
+
+	WRITE "SET $ZGBLDIR=""otherB.gld"" ",!
+	SET $ZGBLDIR="otherB.gld"
+
+	WRITE "VIEW ""STATSHARE"":""DEFAULT"" ",!
+	VIEW "STATSHARE":"DEFAULT"
+	WRITE !
+	do checkRegion()
+	WRITE !
+	WRITE "$VIEW(""STATSHARE"",""DEFAULT""): "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!!
+
+	WRITE "VIEW ""NOSTATSHARE"":""DEFAULT"" ",!
+	VIEW "NOSTATSHARE":"DEFAULT"
+	WRITE !
+	do checkRegion()
+	WRITE !
+	WRITE "$VIEW(""STATSHARE"",""DEFAULT""): "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!!
+
+	quit
+
+checkRegion()
```

Comment:
Can this function also do a "ZWR ^%YGS". This will help us further confirm the consequences of the VIEW "STATSHARE" and VIEW "NOSTATSHARE" commands. Thanks.